docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -336,11 +341,15 @@ Ansible merges different variables set in inventory so that more specific settin
 
 .. rubric:: Footnotes
 
-.. [1] Tasks in each role see their own role's defaults. Tasks defined outside of a role see the last role's defaults.
+.. [1] Tasks in each role see their own role's defaults if specified, or those from the role's dependencies. Tasks defined outside of a role see the last role's defaults.
 .. [2] Variables defined in inventory file or provided by dynamic inventory.
 .. [3] Includes vars added by 'vars plugins' as well as host_vars and group_vars which are added by the default vars plugin shipped with Ansible.
-.. [4] When created with set_facts's cacheable option, variables have the high precedence in the play,
-       but are the same as a host facts precedence when they come from the cache.
+.. [4] When created with set_facts's cacheable option, variables have the high precedence in the play, but are the same as a host facts precedence when they come from the cache.
```

Comment:
I don't understand this note yet. Is this suggestion still accurate?
```suggestion
.. [4] Variables created with set_facts where cacheable=yes have higher precedence in the play than host facts. Variables created with set_facts without cacheable=yes have the same precedence as host facts.
```
Also, how can variables from two different sources have the same precedence . . . if they are different, which one wins?

Reply:
fwiw, I didn't change anything with the wording here, I just removed the line wrap.  Basically `set_facts` without `cacheable=yes` create and overwrite host facts, as they exist in the same "dict".

Reply:
I understand this a bit better after our conversation today. I'll close this suggestion and possibly put in a slightly different one.